tests/test_strax_time_selection.py

```diff
@@ -0,0 +1,60 @@
+"""

```

Comment:
Maybe add this text to your assert message.

Reply:
Thanks, the assert statement should simply never fail it's not really the test itself. Nonetheless added a sting just above the - would be failing - line